index.js

```diff
@@ -176,6 +176,45 @@ if(options.index){
   
   
       })
+    } else if (path.extname(file) == ".md"){
+      console.log("md file inputted!")
+      const folderName = 'dir';
+  
+      const htmlFile = fs.readFileSync(`${__dirname}/index.html`)
+  
+      filenameWithoutExt = path.parse(file).name; // The name part of the file (EX: name.txt => name)
+  
+      fs.writeFileSync(`${process.cwd()}/${folderName}/${filenameWithoutExt}.html` , htmlFile);  
+      
+      fs.readFile(file , {encoding:'utf8', flag:'r'},
+      function(err, data) {
+        if(err)
+        console.log(err);
+       
+
+        var editedText = data.split(/\r?\n\r?\n/)
+        .map((para) =>
+          para
+            .replace(/^# (.*$)/gim, '<h1>$1</h1>')
+            .replace(/^## (.*$)/gim, '<h2>$1</h2>')
+            .replace(/^--- (.*$)/gim, '<hr/>')
+        )
+        .join('\n');
+    
+        let title = editedText.split("</p>")[0].split(">" , 2)[1]; // getting the title of the text

```

Comment:
This is causing issues when the md file has any of the formatting from md in the first line. It will try to parse it AFTER the modification and is missing a closing bracket. Working on it right now! :)